### DISCUSSION TITLE
Linux Mint Ec_Sys not working

### DISCUSSION THREAD

**Discussion by Wy477-P** [YoCodingMonster/OpenFreezeCenter#159] [Q&A]
I'm very new to linux and have no experience with it, so I have been very frustrated for the past week trying to get something to even make my laptop not overheat. Every time i try something it asks for ec_sys which the kernel says it has but when i try to add write support and update grub, it says it's not there. I have tried all the troubleshooting steps and still no luck. the furthest i got was when i first ran the at_startup file and a window flashed before refusing to ever open again. I'm completely stumped and don't know what to do at this point other than change my os to something that works out of the box which would be a pain as i spent a while moving my files to linux mint.

**Comment by YoCodingMonster** [comment]
Please try the new release of both OpenFreezeCenter and OpenFreezeCenter-Lite and report back if the applications are working.
Also the installation steps have been simplified and well documented and i am again active on github, so dont worry, just keep posting the issues and i will guide you through it!
